extensions/ql-vscode/package.json

```diff
@@ -501,6 +501,11 @@
         "command": "codeQL.copyVersion",
         "title": "CodeQL: Copy Version Information"
       },
+      {
+        "command": "codeQL.runLocalQueryFromQueriesPanel",
```

Comment:
We still want this command to be prefixed with `codeQLQueries` instead of `codeQL`, since its a command that is scoped to a specific CodeQL-controlled panel (instead of being generally available in the file editor/command palette). 

(This would match the other commands in `"view/item/context"` too 😎 )

Reply:
Ah, yes 🤦‍♀️